lib/anubis_test.go

```diff
@@ -36,7 +37,7 @@ func spawnAnubis(t *testing.T, opts Options) *Server {
 
 func TestCookieSettings(t *testing.T) {
 	pol := loadPolicies(t, "")
-	pol.DefaultDifficulty = 0
+	pol.DefaultDifficulty = 4
```

Comment:
This probably needs a dedicated test, EG:

```go
func TestCVE2025_24369(t *testing.T)
```

Also please check the email listed on your github profile :)